packages/discovery-react-components/src/components/SearchResults/components/Result/Result.tsx

```diff
@@ -113,7 +113,12 @@ export const Result: React.FunctionComponent<ResultProps> = ({
   if (result) {
     documentId = result.document_id;
   }
-  const title = get(result, resultTitleField);
+
+  let title = get(result, resultTitleField);
+  if (title && title.length > 100) {
+    title = title.substring(0, 100) + '...';
```

Comment:
i would use an ellipsis character here -> `…`

Reply:
also is there any way to do this with CSS instead? `text-overflow: ellipsis` should accomplish what we need here